proposals/nnnn-remote-control-seat.md

```diff
@@ -0,0 +1,233 @@
+# Remote Control - Seat
+
+* Proposal: [SDL-NNNN](nnnn-remote-control-seat.md)
+* Author: [Zhimin Yang](https://github.com/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+SDL remote control allows a user to control certain modules of the vehicle. This proposal allows mobile applications control powered seats within a vehicle (while the vehicle is in parking). A mobile application can send a remote control command to adjust seat positions, turn on or off cooled/heated seat, or change massage settings. The application can also get the status of each seat as well as receive notifications once the status of any seat changes.
+
+## Motivation
+
+To extend SDL-RC functionality and add a "SEAT" remote-control module of the vehicle. 
+
+## Proposed solution
+
+Add a `SEAT` RC module, the capabilities of each seat, and the related control data type to `Mobile_API.xml` and `HMI_API.xml`
+
+| Control Item | Value Range |Type | Comments |
+| ------------ |------------ | ------------ | ------------ |
+| Seat Heating Enabled | true, false | Get/Set/Notification | Indicates whether heating is enabled for a seat |
+| Seat Cooling Enabled | true, false | Get/Set/Notification | Indicates whether cooling is enabled for a seat |
+| Seat Heating  level | 0-100% | Get/Set/Notification | Level of the seat heating |
+| Seat Cooling  level | 0-100% | Get/Set/Notification | Level of the seat cooling |
+| Seat Horizontal Positon | 0-100% | Get/Set/Notification | Adjust a seat forward/backward, 0 means the nearest position to the steering wheel, 100% means the furthest position from the steering wheel |
+| Seat Vertical Position | 0-100% | Get/Set/Notification | Adjust seat height (up or down) in case there is only one actutator for seat height, 0 means the lowest position, 100% means the highest position|
+| Seat-Front Vertical Position | 0-100% | Get/Set/Notification | Adjust seat front height (in case there are two actutators for seat height), 0 means the lowest position, 100% means the highest position|
+| Seat-Back Vertical Position | 0-100% | Get/Set/Notification | Adjust seat back height (in case there are two actutators for seat height), 0 means the lowest position, 100% means the highest position|
```

Comment:
Please change "actutators" to "actuators"

Reply:
updated, thank you